sdk/agrifood/agrifood-farming-rest/package.json

```
@@ -95,8 +96,9 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^2.0.1",
     "@azure/test-utils": "^1.0.0",
-    "@azure-tools/test-recorder": "^1.0.0",
-    "@microsoft/api-extractor": "^7.31.1",
+    "@azure-tools/test-credential": "~1.0.0",
+    "@azure-tools/test-recorder": "~2.0.0",
+    "@microsoft/api-extractor": "7.18.11",
```

Please keep the version to the latest one

```suggestion
    "@microsoft/api-extractor": "^7.31.1",
```